pkg/cmd/target/target.go

```diff
@@ -54,6 +51,22 @@ func NewCmdTarget(f util.Factory, o *TargetOptions) *cobra.Command {
 	return cmd
 }
 
+type cobraRunFunction = func(cmd *cobra.Command, args []string) error
+
+func runCmdTargetWrapper(f util.Factory, o *TargetOptions) cobraRunFunction {
```

Comment:
Use `base.WrapRunE` instead and implement Run on TargetOptions.
```go
// Run does the actual work of the command.
func (o *Options) Run(util.Factory) error {
	return runCmdTarget(f, o)
}
```

Reply:
done